include/pipeline.h

```diff
@@ -47,13 +47,13 @@ namespace grppi {
 \param trasnform_ops Transformation operations for each stage.
 */
 template <typename Execution, typename Generator, typename ... Transformers,
-          requires_execution_supported<Execution> = 0>
+          requires_execution_supported<std::decay_t<Execution>> = 0>
 void pipeline(
     const Execution & ex, 
     Generator && generate_op, 
     Transformers && ... transform_ops) 
 {
-  static_assert(supports_pipeline<Execution>(),
+  static_assert(supports_pipeline<std::decay_t<Execution>>(),
```

Comment:
Is this static_assert useful given the requires clause?

Reply:
The requires clause in the template parameters only checks if the Execution is an actual execution policy object, while the static_assert checks if the execution policy implements the pipeline pattern.

Reply:
Agreed! Thanks!